### ISSUE TITLE
[Info] Availability of 32bit plugin

### ISSUE THREAD

**Issue by pidgeon777** [Ekopalypse/NppVLexer#1]
Hello, I was wondering if the 32bit edition of the plugin is available somewhere.

**Comment by Ekopalypse** [comment]
No, the reason is the different handling of [thiscall](https://docs.microsoft.com/en-us/cpp/cpp/thiscall?view=msvc-170) on x86 and x64. I haven't bothered to find out if there is a way for C and gcc to make this work. If you happen to know what needs to be done, I'm willing to test and see if this can be implemented, but it's unlikely that I'll spend any time myself figuring out what needs to be done since I no longer use an x86 OS.

**Comment by Ekopalypse** [comment]
@pidgeon777 - just in case that you aren't aware, there is an [udl for V](https://github.com/vlang/notepad-plus-plus)

**Comment by Ekopalypse** [comment]
32bit version available in v.0.6
